apps/meteor/ee/app/livechat-enterprise/server/business-hour/Helper.ts

```diff
@@ -10,7 +10,7 @@ const getAllAgentIdsWithoutDepartment = async (): Promise<string[]> => {
 	).map((dept: any) => dept.agentId);
 	const agentIdsWithoutDepartment = (
 		await Users.findUsersInRolesWithQuery(
-			'livechat-agent',
+			['livechat-agent'],
```

Comment:
So, this is a "current" bug, or does the "findUsersInRolesWithQuery" accepts string | string[] ? 👀

Reply:
No, using non-list has been deprecated, don't know how long. I just switched all singles to lists.

Reply:
naice